src/Aspire.Hosting.Azure.CognitiveServices/AzureOpenAIDeployment.cs

```diff
@@ -31,10 +35,16 @@ public class AzureOpenAIDeployment(string name, string modelName, string modelVe
     /// <summary>
     /// Gets the name of the SKU.
     /// </summary>
-    public string SkuName { get; } = skuName;
+    /// <value>
+    /// The default value is <inheritdoc cref="DefaultSkuName"/>.
+    /// </value>
+    public string SkuName { get; set; } = DefaultSkuName;
 
     /// <summary>
     /// Gets the capacity of the SKU.
     /// </summary>
-    public int SkuCapacity { get; } = skuCapacity;
+    /// <value>
+    /// The default value is <inheritdoc cref="DefaultSkuCapacity"/>.
```

Comment:
This is maybe too much here, if anyone thinks so that I will remove the constants and just put the values in the property and the comment directly.